src/app/shared/entry.ts

```diff
@@ -125,11 +125,12 @@ export abstract class Entry implements OnDestroy {
     }
   }
 
-  protected isAppTool(url: String): boolean {
-    if (url.includes('/containers/github.com')) {
-      return true;
-    }
-    return false;
+  protected isAppTool(url: string): boolean {
+    return url.includes('/containers/github.com');
```

Comment:
Let's hope nobody ever creates an app tool named `notebooks` :) Or it's their GitHub userid. Not worth worrying about.